settings.gradle.kts

```diff
@@ -1 +1,5 @@
+plugins {
+    id("org.gradle.toolchains.foojay-resolver-convention") version "0.4.0"
```

Comment:
> Gradle 7.6 introduced [arbitrary toolchain repositories](https://docs.gradle.org/8.0-rc-1/userguide/toolchains.html#sub:download_repositories). In Gradle 8.0, there is no longer a default toolchain provisioner.

https://docs.gradle.org/8.0-rc-1/release-notes.html

Reply:
Although the project's standard gradle is still 7.6, I assume you're doing this for `release-candidate` in the test matrix...

Reply:
>I assume you're doing this for release-candidate in the test matrix...

Correct 👍 We could wait with this change for a while. I assumed this could be considered future-proofing and the upcoming migration to Gradle 8.0 will be simpler. 
If you prefer, I can try to do a similar thing as we did with Android Gradle Plugin RCs and to test them only the main branch 👀